.github/workflows/functional_all_db.yml

```diff
@@ -150,6 +162,9 @@ jobs:
       OrchardCore__DatabaseProvider: "SqlConnection"
     steps:
     - uses: actions/checkout@v3
+    - uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '7.0.x'
```

Comment:
shouldn't it be the default already?

Reply:
Yes, but as i remember the problem was just that NodeJs was not finding the dotnet command path, and that the `setup-dotnet@v3` action updates correctly the PATH env variable. So if dotnet is already installed, maybe it just updates the PATH env variable.

Would need to dig in more, if I have time I will try to just update the PATH env variable.

Reply:
My question was about the `dotnet-version`, it shouldn't be necessary. This way when net8.0 is GA we won't have to change this line

Reply:
Okay will try it